lectures/13/theory.tex

```diff
@@ -0,0 +1,67 @@
+% Lecture date: 05/06/2019, part one and two
+% Authors: Ambuj Ojha, Kuixian Zhu, Yang Stelmach
+% Time interval: entire class
+\chapter{Self-Supervised Learning}
+
+\section{Introduction and Why Self-supervised Learning}
+
+The success story of computer vision can be viewed as the success story of supervision. For example, over the past decade, there has been continuous improvement in classification results on the ImageNet Challenge. 
+
+Moreover, it has been shown that features from networks that are pre-trained on large datasets like ImageNet could be used for a variety of different downstream tasks. There is a general agreement that the 'recipe' for a computer vision supervised task is to:
+
+\begin{itemize}
+    \item Pre-train the model (usually a ConvNet) on a large supervised dataset
+    \item Collect a dataset of supervised images
+    \item Keep training the model
+\end{itemize}
+
+However, for a variety of data, such as video and medical images, it is hard to scale the labeling to the size of data we generate. Sadly, the labeled data that we are able to utilize is only an incredibly small portion of the data in our real world. Unlabelled internet images are several orders of magnitudes larger than labelled images. For instance, it would take 22 human years to label the remainder of ImageNet data. Thus, it is important for us to develop a 'self-automatic' process that can obtain labels from the data itself, which is the common definition of self-supervised learning.
+
+One of the most successful examples of self-supervised learning is the representation learning in NLP. From  Word2vec (Mikolov et al.) to BERT (Devlin et al.), it has been shown that self-supervised learning can be helpful in getting high-quality sentence/word representations.
+
+Indeed, self-supervised learning is helpful because it:
+
+\begin{itemize}
+    \item Helps us learn using observations and interactions
+    \item Does not require exhaustive annotation of concepts
+    \item Leverages multiple modalities or structures in the domain
+\end{itemize}
+
+\section{Self-supervised Learning in Computer Vision}
+
+For computer vision, below three things are most popular:
+\begin{itemize}
+    \item Using images
+    \item Using video
+    \item Using video and sounds
+\end{itemize}{}
+
+\subsection{Using images}
+Images always have an underlying spatial structure. This regularity is also the reason why convolutional network really works with images. 
+
+\begin{itemize}
+    
+\item One of the first successful applications of self-supervised learning using images is in the task of finding the relative position of patches. The idea is to sample two patches in an image, for example - in the image of the cat in Fig.\ref{fig:1} one (blue patch) is the nose of the cat and the other (red patch) is on the eye of the cat. 
+
+\begin{figure}[h!]
+  \includegraphics[width=\linewidth]{RelativePosition.jpg}
+  \caption{Relative Position of patches}
+  \label{fig:1}
```

Comment:
This is not an acceptable label.

Reply:
Can I still fix this?

Reply:
Yes, send a new PR with a change of label.
Just one commit. Not the entire lecture notes.

Reply:
Also, see below. I think you need to rename `13` to `13-a` since others have taken care of `13-b`.
Or I'm not understanding what is going on.

Reply:
Hi Alf, dumb question - how can I commit just this part, rather than the entire lecture notes?

Reply:
You have to pull the latest master, and rebase on top of it.